components/channel_header_dropdown/mobile_channel_header_dropdown.tsx

```diff
@@ -79,7 +81,7 @@ export default class MobileChannelHeaderDropdown extends React.PureComponent {
                     defaultMessage='Channel Menu'
                 >
                     {(ariaLabel) => (
-                        <Menu ariaLabel={ariaLabel}>
+                        <Menu ariaLabel={`${ariaLabel}`}>
```

Comment:
why did you change it, the current version doesn’t change anything
```suggestion
                        <Menu ariaLabel={ariaLabel}>
```

Reply:
Menu expects ariaLabel to be string but ariaLabel is actually ReactNodeArray not string.
So ariaLabel should be cast to string.

Reply:
let's use `formatMessage` instead of `formattedMessage` instead of this hack,

Reply:
this is a very good suggestion

Reply:
I used `formatMessage` instead of `FormattedMessage`